sros2/test/test_policy_to_permissions.py

```diff
@@ -23,6 +24,20 @@
 )
 
 
+if 'XML_CATALOG_FILES' not in os.environ:
+    os.environ['XML_CATALOG_FILES'] = os.path.join(
+        ament_index_python.get_package_share_directory('sros2'),
+        'xml_cache',
+        'xhtml-cache.xml'
+    )
+elif 'xml_cache/xhtml-cache.xml' not in os.environ['XML_CATALOG_FILES']:
+    os.environ['XML_CATALOG_FILES'] += ' ' + os.path.join(
```

Comment:
It's not clear to me what delimiter lxml supports, be it a space or `:` or if lists of paths are supported at all. Do you know of an example somewhere? I only see it referenced here: http://xmlsoft.org/catalog.html

Reply:
Oh, from here: https://stackoverflow.com/a/8391738/2577586
> Note that entries in XML_CATALOG_FILES are space-separated URLs. You can use Python's pathname2url and urljoin (with file:) to generate the URL from a pathname.

Do we need to prefix the path with `file://`?

Reply:
I don't know.  I tested it manually by setting my XML_CATALOG_FILES environment variable to `/etc/xml/catalog` and verified that the test still passes when it appends to the existing environment variable.

On [this xmlsoft.org page](http://xmlsoft.org/catalog.html) it doesn't mention anything about file://

```
The user can change the default catalog behaviour by redirecting queries to its own set of catalogs, this can be done by setting the XML_CATALOG_FILES environment variable to a list of catalogs, an empty one should deactivate loading the default /etc/xml/catalog default catalog
```

Reply:
@ruffsl It seems to work OK. . .  I can pass the test without an internet connection, and my final XML_CATALOG_FILES environment variable is:

```
/etc/xml/catalog file%3A///sros2/install/sros2/share/sros2/xml_cache/xhtml-cache.xml
```